src/analysis/special_functions/pow.lean

```diff
@@ -1493,4 +1493,10 @@ lemma measurable.ennreal_rpow_const {α} [measurable_space α] {f : α → ennre
   measurable (λ a : α, (f a) ^ y) :=
 hf.ennreal_rpow measurable_const
 
+lemma ae_measurable.ennreal_rpow_const {α} [measurable_space α] {f : α → ennreal}
+  {μ : measure_theory.measure α} (hf : ae_measurable f μ) {y : ℝ} :
+  ae_measurable (λ a : α, (f a) ^ y) μ :=
+⟨λ a, (hf.mk f a)^y, hf.measurable_mk.ennreal_rpow_const,
+  eventually_eq.fun_comp hf.ae_eq_mk (λ (a : ennreal), a ^ y)⟩
```

Comment:
Isn't it `measurable.comp_ae_measurable _ hf`?

Reply:
Sure. Both proofs are straightforward, so I used one or the other randomly, I guess.

Reply:
I changed this proof and other ones to the shorter version.